extension/src/setup/studio.ts

```diff
@@ -108,6 +109,8 @@ export class Studio extends Disposable {
   }
 
   public async requestStudioTokenAuthentication() {
+    this.studioAccessTokenUriHandler?.dispose()
```

Comment:
[C] personal preference but I always set class members to `undefined` after disposing.

Reply:
could create a `resetAccessTokenUriHandler` method

Reply:
Done!